### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by ghoneycutt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-- tests/test.sh
+  - bundle exec rubocop
+  - tests/test.sh
+
 sudo: false
+
 rvm:
-- 2.2
+  - 2.1
+  - 2.3
```
Comment (rnelson0) [inline .travis.yml]: I think you need `2.3.1` or similar, they stopped doing just `x.y` and required `.z` at some point.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ Simple tool to validate and lint `metadata.json` files in Puppet modules as
 recommended in Puppet Forge style guidelines from [Puppet forge metadata style
 quide](https://docs.puppetlabs.com/puppet/latest/reference/modules_publishing.html#write-a-metadatajson-file)
 
+## Compatibility
+
+metadata-json-lint is compatible with Ruby versions 2.1.x and 2.3.x.
```
Comment (rnelson0) [inline README.md]: Is there a reason we would skip 2.2.x? It wouldn't add a lot of test time to add it to the matrix.

File: tests/test.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -x
```
Comment (rnelson0) [inline tests/test.sh]: This is VERY verbose. If we feel it needs to output something, it should provide very minimal output, one line per test, indicating test name and success or failure at best. `-x` mode is way too verbose to find actual failures ([~250 lines](https://travis-ci.org/voxpupuli/metadata-json-lint/jobs/172084400#L178-L425)).

File: README.md (hunk 2 above)
Comment (ghoneycutt) [inline README.md]: 2.1.9 is the current ruby version in the AIO and 2.3.1 has been announced as the next version. Is there a platform that ships with 2.2.x? If so, let's definitely add it.

I also need to add 2.0.0 as that's the default for EL7

File: tests/test.sh (hunk 3 above)
Comment (ghoneycutt) [inline tests/test.sh]: +1

suggest using `-x` until someone makes the output better as you suggested. Would rather have `-x` than the current state of no output.

File: README.md (hunk 2 above)
Comment (ghoneycutt) [inline README.md]: took another look and it was a simple fix, which I have incorporated.

File: tests/test.sh (hunk 3 above)
Comment (ghoneycutt) [inline tests/test.sh]: took another look and it was a simple fix, which I have incorporated.

File: tests/test.sh [hunk 4]
```diff
@@ -21,6 +21,8 @@ test() {
   RESULT=$?
   if [ $RESULT -ne $expect ]; then
       fail "Failing Test '${name}' (bin)"
+  else
+      echo "Success testing '${name}' (bin)"
```
Comment (rnelson0) [inline tests/test.sh]: Is it too nit picky to have it say 'Successful Test', to better match the failure comment?

**Comment by ghoneycutt** [comment]
Thanks!

**Pull request merged**